lib/control/validation/content.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const HTTP_BAD_REQUEST = 400;
+
+const M_INSTANCE_ID = /^i-[a-f0-9]{8,}$/;
+const M_AMI_ID = /^ami-[a-f0-9]{8,}$/;
+const M_ACCOUNT_ID = /^[0-9]{12,}/;
+const M_REGION = /^[a-z]{2,}-[a-z]{4,}-[1-2]$/;
+
+exports.document = function(req,res,next) {
+
+  req.document = JSON.parse(req.body.document);
```

Comment:
The first check (the valid.js module) checks to make sure that the document is json. which is why there isn't a try catch block here.